beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -681,3 +700,12 @@ func attsForCrosslink(state *pb.BeaconState, crosslink *pb.Crosslink, atts []*pb
 	}
 	return crosslinkAtts
 }
+
+// totalActiveBalance returns the combined balances of all the active validators.
+//
+// Spec pseudocode definition:
+//	def get_total_active_balance(state: BeaconState) -> Gwei:
+//    return get_total_balance(state, get_active_validator_indices(state, get_current_epoch(state)))
+func totalActiveBalance(state *pb.BeaconState) uint64 {
```

Comment:
maybe this should be under the helpers package ? I have seen it in your other PR too

Reply:
yeah... it used to be in general helpers package, I moved it to `epoch_processing` because it is only being used during `epoch_processing`. The spec did the same thing, `get_total_balance` is contained under `epoch_processing` section 😅